ports/eigen3/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "eigen3",
+  "version-string": "3.4.0",
```

```suggestion
  "version-semver": "3.4.0",
```
This should be [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-semver).